### ISSUE TITLE
Soul of Colossus health bonus is sometimes missing on world startup

### ISSUE THREAD

**Issue by atealein** [STS15/Fargos-Talismans#27]
### Modpack

Yes

### Neoforge Version

21.1.66

### Mod Version

FTB Evolution Beta

### Describe the issue

Soul of Colossus is providing +400% on the max health the player has. However, I've noticed that sometimes (unsure yet the trigger) on loading the world, the soul of colossus health bonus is missing from the attributes list and will not appear - regardless if the charm is removed, taken out of inventory, placed back in again. Simply rejoining the world also doesn't fix it. So far the two times I've noticed this happen, I've had to do an entire instance shut down and then start it up from the mod launcher fresh for it to be fixed (maybe it is overkill, but not sure what else could be done). This is on a Multiplayer server, so maybe the issue is client side?

**Comment by sanomi78** [comment]
I don't think this bug has been solved. I just updated from 2.1.3 to 2.1.5 and it still resets my hearts after i log back in and i have to eat much food to get back to full blue hearts.

**Comment by STS15** [comment]
![image](https://github.com/user-attachments/assets/872978b9-d344-425a-9bf4-202713a7d44a)

**Comment by Carlcamp** [comment]
im getting the health issue with the soul of minecraft as well if you die, only fix is to reboot the server.
